src/views/DidUpgradeExplainer/DidUpgradeExplainer.tsx

```diff
@@ -165,7 +165,7 @@ export function DidUpgradeExplainer({ identity }: Props): JSX.Element | null {
                   onChange={handleChange}
                   className={styles.select}
                 />
-                {t('view_DidUpgradeExplainer_euro', [euroCost])}
+                {euroCost && t('view_DidUpgradeExplainer_euro', [euroCost])}
```

Comment:
So if `euroCost` is undefined we show the radio input without any label?

Reply:
Good point, let’s rather show nothing until the data arrives